x-pack/plugins/fleet/public/components/new_enrollment_key_modal.tsx

```diff
@@ -86,15 +94,18 @@ export const NewEnrollmentTokenModal: React.FunctionComponent<Props> = ({
       <form onSubmit={form.onSubmit}>
         <EuiFormRow
           label={i18n.translate('xpack.fleet.newEnrollmentKey.nameLabel', {
-            defaultMessage: 'Name',
+            defaultMessage: 'Token name',
           })}
         >
-          <EuiFieldText
-            required={true}
-            name="name"
-            autoComplete="off"
-            {...form.apiKeyNameInput.props}
-          />
+          <EuiFlexItem>
+            <EuiFieldText
+              name="name"
+              autoComplete="off"
+              placeholder="Enter a token name"
```

Comment:
We probably want to translate that you can use `i18n.translate()` here

Reply:
okay, I wasn't sure where is it needed, as I don't see it used everywhere.
is there a way to find out which components translate out of the box?

Reply:
No components translate out of the box, so if there is not translation we missed something previously

Reply:
updated